server/src/controllers/Events/resolver.ts

```diff
@@ -719,22 +700,11 @@ ${unsubscribeOptions}`,
     await prisma.$transaction([
       prisma.events.update({
         where: { id },
-        data: { tags: { deleteMany: {} } },
+        data: {},
       }),
       prisma.events.update({
         where: { id },
-        data: {
-          tags: {
-            create: getUniqueTags(data.tags).map((tagName) => ({
-              tag: {
-                connectOrCreate: {
-                  create: { name: tagName },
-                  where: { name: tagName },
-                },
-              },
-            })),
-          },
-        },
+        data: {},
```

Comment:
Shouldn't these be deleted as well? I amn't sure, but now they don't update anything